states/nginx/files/api.creativecommons.engineering

```diff
@@ -0,0 +1,36 @@
+# vim: ft=nginx:
+
+
+server {
+    listen 80;
+    listen [::]:80;
+    server_name
+        api.creativecommons.engineering
+        api-dev.creativecommons.engineering;
+    access_log off;
+    # Enable letsencrypt authorization
+    location /.well-known {
+        default_type "text/plain";
+        root /var/www/html;
+    }
+    # Redirect everything else
+    # (will purposely break API calls, but bring humans to current documentation)
+    location / {
+        return 301 https://api.openverse.engineering/?referrer=creativecommons.org;
```

Comment:
We're not doing anything to read this `referrer` query parameter on the API, but good to know it's present if we decide to.